src/main/java/com/backend/soullive_a/entity/ModelImageKeyword.java

```diff
@@ -0,0 +1,29 @@
+package com.backend.soullive_a.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+
+public class ModelImageKeyword {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long modelImageKeywordId;
+
+    @ManyToOne
+    @JoinColumn(name = "model_id") //비식별인데..
```

Comment:
```suggestion
    @JoinColumn(name = "MODEL_INTRODUCTION_ID")
```
"MODEL_INTRODUCTION_ID" 처럼 대문자로 하는 걸로 통일했습니다..! 그리고 필요없는 주석제거 해주세용

Reply:
알겠습니